prototype_source/pt2e_quant_ptq_static_x86_inductor.rst

```diff
@@ -0,0 +1,188 @@
+PyTorch 2.0 Export Post Training Static Quantization with X86 Backend through Inductor
+========================================================================================
+
+**Author**: `Leslie Fang <https://github.com/leslie-fang-intel>`_, `Weiwen Xia <https://github.com/Xia-Weiwen>`_, `Jiong Gong <https://github.com/jgong5>`_, `Jerry Zhang <https://github.com/jerryzh168>`_
+
+Prerequisites
+^^^^^^^^^^^^^^^
+
+-  `PyTorch 2.0 Export Post Training Static Quantization <https://pytorch.org/tutorials/prototype/pt2e_quant_ptq_static.html>`_
+-  `TorchInductor and torch.compile concepts in PyTorch <https://pytorch.org/tutorials/intermediate/torch_compile_tutorial.html>`_
+
+Introduction
+^^^^^^^^^^^^^^
+
+This tutorial introduces the steps for utilizing the PyTorch 2.0 Export Quantization flow to generate a quantized model customized
+for the x86 inductor backend and explains how to lower the quantized model into the inductor.
+
+- New Quantization 2.0 flow uses the PT2 Export to capture the model into a graph and perform quantization transformations on top of the ATen graph. This approach is expected to have significantly higher model coverage, better programmability, and a simplified UX.
+- TorchInductor is the new compiler backend that compiles the FX Graphs generated by TorchDynamo into optimized C++/Triton kernels.
+
+This flow of quantization 2.0 with Inductor mainly includes 3 steps:
+
+- Step 1: Capture the FX Graph from the eager Model based on the `torch export mechanism <https://pytorch.org/docs/main/export.html>`_.
+- Step 2: Apply the Quantization flow based on the captured FX Graph, including defining the backend-specific quantizer, generating the prepared model with observers,
+  doing the prepared model's calibration, and converting the prepared model into the reference quantized model.
+- Step 3: Lower the reference quantized model into inductor with the API ``torch.compile``.
+
+The high-level architecture of this flow could look like this:
+
+::
+
+    float_model(Python)                               Input
```

Comment:
can you change Input to `Example Input` to make it clearer?

Reply:
Thanks and changed.